qa/tasks/ceph_manager.py

```diff
@@ -1540,11 +1540,13 @@ def tmp(x):
         self.cephadm = cephadm
         self.testdir = teuthology.get_testdir(self.ctx)
         # prefix args for ceph cmds to be executed
-        pre = ['adjust-ulimits', 'ceph-coverage',
-               f'{self.testdir}/archive/coverage']
-        self.CEPH_CMD = ['sudo'] + pre + ['timeout', '120', 'ceph',
-                                          '--cluster', self.cluster]
-        self.RADOS_CMD = pre + ['rados', '--cluster', self.cluster]
+        self.pre = ['adjust-ulimits', 'ceph-coverage',
+                    f'{self.testdir}/archive/coverage']
+        # keeping CEPH_CMD for reference
+        # this is now replaced with get_ceph_cmd()
+        self.CEPH_CMD = ['sudo'] + self.pre + ['timeout', '120', 'ceph',
+                                               '--cluster', self.cluster]
```

Comment:
I'd rather see this removed than kept for reference.

Please update vstart_runner.py too.

Reply:
okay, I've removed `self.CEPH_CMD` from `ceph_manager.py` and `vstart_runner.py`
also, since `vstart_runner.py` doesn't use any timeouts for `ceph` commands, the overloaded `get_ceph_cmd()` in `vstart_runner.py` just returns the path to the local `ceph` command-line utility
also, `class LocalCephManager` derives out of `class CephManager` so the correct overloaded `get_ceph_cmd()` will get invoked